### ISSUE TITLE
Fake death in City of Threads

### ISSUE THREAD

**Issue by swixfort** [Vloer/keycount#19]
When someone is being entagled by the patrols in City of Threads, it is counted (by KeyCount) as a death. It is no death though and you only get ported back to the start of the section and lose some health.

**Comment by Vloer** [comment]
I don't think this is actually fixable easily since for some reason (apparently) the ingame combat log counts it as a death. Will try to look into it
